### PR TITLE
Simplified Participant View

### PR DESCRIPTION
**Pull request by antonmc**

### CODE REVIEW

File: iOS/rainbow/Controller/Leaderboard/ParticipantViewController.swift [hunk 1]
```diff
@@ -15,7 +15,7 @@ class ParticipantViewController: UIViewController {
     @IBOutlet weak var timeElapsedLabel: UILabel?
     @IBOutlet weak var scavagingSinceLabel: UILabel?
     @IBOutlet weak var progressLabel: UILabel?
-    @IBOutlet weak var progressSlider: UISlider?
+    @IBOutlet weak var button:UIButton?
```
Comment (dokun1) [inline iOS/rainbow/Controller/Leaderboard/ParticipantViewController.swift]: We should make this name more expressive. Can we call it `leaderboardButton` instead? Also, please add a space between the right side of the colon and the class type.

File: iOS/rainbow.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -664,7 +661,6 @@
 				5305A1AC20A219380040A9A9 /* Logger.swift in Sources */,
 				53C6D18B20ADBB10008DF112 /* ProjectRainbowModel_1753554316.mlmodel in Sources */,
 				5305A1BA20A219380040A9A9 /* PersistableExtension.swift in Sources */,
-				A2B2CB2820AE117000A6F585 /* UIImageExtension.swift in Sources */,
```
Comment (dokun1) [inline iOS/rainbow.xcodeproj/project.pbxproj]: Just to be sure - we saw that this was a deleted class. We don't actually need this, right?

File: iOS/rainbow/Controller/Leaderboard/ParticipantViewController.swift [hunk 3]
```diff
@@ -48,9 +48,13 @@ class ParticipantViewController: UIViewController {
             if let savedFoundObjects = savedEntry.objects {
```
Comment (dokun1) [inline iOS/rainbow/Controller/Leaderboard/ParticipantViewController.swift]: If we don't show progress on this view anymore, we can get rid of the objects we use to keep track of the found objects thus far. Let's delete these, as well as the `config` and `foundObjects` array above it please.

File: iOS/rainbow/View/Base.lproj/Main.storyboard [hunk 4]
```diff
@@ -321,136 +321,163 @@
...
-                                <rect key="frame" x="14" y="374" width="347" height="30"/>
-                                <constraints>
-                                    <constraint firstAttribute="height" constant="29" id="iHD-VQ-97a"/>
-                                </constraints>
-                            </slider>
-                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="X0Q-em-HSV">
-                                <rect key="frame" x="16" y="543" width="343" height="42"/>
-                                <color key="backgroundColor" red="0.0" green="0.61176470589999998" blue="0.87450980389999999" alpha="1" colorSpace="calibratedRGB"/>
+                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" translatesAutoresizingMaskIntoConstraints="NO" id="j0j-Xz-mVk">
```
Comment (dokun1) [inline iOS/rainbow/View/Base.lproj/Main.storyboard]: I've tried this build and there is an auto-layout issue with some conflicting constraints. Please make the other changes and I'll take care of the auto-layout issues afterwards on the same branch.

File: iOS/rainbow.xcodeproj/project.pbxproj (hunk 2 above)
Comment (antonmc) [reply iOS/rainbow.xcodeproj/project.pbxproj]: we don't need that class

**Pull request merged**